src/decisionengine/framework/taskmanager/TaskManager.py

```diff
@@ -347,7 +308,6 @@ def run_cycle(body, message):
                 self.data_block_put(data, header, self.data_block_t0)
                 self.logger.info(f"Source {module_name} data block put done")
 
-                self.logger.info(f"Message on bus: {body}")
```

Comment:
I'd be tempted to increment this to `debug` rather than remove the logging.

Reply:
Hmm...not sure about this one.  I don't *think* we've logged data before, and I'm a little leery of the data volume.  Maybe we should log it if there's an error inserting it into the data block?

Reply:
> Maybe we should log it if there's an error inserting it into the data block?

That works for me